Automattic/wp-calypso
# Refactor/make Swipeable component

This adds a swipeable component and integrates it into the home section.  

#### Changes proposed in this Pull Request

This PR refactors https://github.com/Automattic/wp-calypso/pull/58533
by extracting a new swipeable component. That enables horizontal swiping. 

#### Testing instructions

Visit /home/example.com 
Notice that you are able to swipe the 2 dot components are now as expected.
You are also able to use the arrows and the dot components (slideshows) behave as before.

Visit /devdocs/design/swipeable
and you are able to swipe though the different pages as expected.

Visit /devdocs/design/dot-pager
and you are able to swipe through the different pages as expected.

Try this across different browsers. 
- iOS Simulator ( Launch xcode )  - you are not able to login to calypso.localhost or any other site other the wordpress.com - but you can test it using the devdocs url.
- Firefox, Chrome, Edge, Safari 

Try it also using the dev tools and resize the window to different widths. 

See

https://user-images.githubusercontent.com/115071/145516869-990e36c5-236a-47a4-a19a-5b96d0a0270b.mp4
